CompanyAddress.md

```diff
@@ -7,124 +7,73 @@ _______
 - [Summary](#summary)
 - [Request structure](#request-structure)
 - [Response structure](#response-structure)
-- [Sample response](#sample-response)
 - [Versioning](#versioning)
 
 ## Summary
 
-The Company Address Service takes a request containing a company name, latitude and longitude and returns a response describing a normalized geographic location.
+The Company Address Service takes a request containing a company id, other company details as described in the below request structure and returns a response containing company address. CompanyAddress is available at 
+`/core/geography/companyaddress`.
```

Comment:
I'd also like to know what effect on the results adding a branch name would have on the final results.

Reply:
Could we provide the full URL with host? If we had a sister company like Broadbean start using this they may not be familiar with what all careerbuilder service endpoints look like and would be kind of lost when we point them to this page. You could also link to the ReadMe which provides more comprehensive general information that all our services share.

Reply:
so the nomalized name is of master company and  address will be that of branch if given ,otherwise the location information in the request should be exact match...if we get multiple matches will return empty response.